packages/meteor/debug.js

```diff
@@ -39,6 +39,11 @@ Meteor._debug = function (/* arguments */) {
 
         if (allArgumentsOfTypeString)
           console.log.apply(console, [Array.prototype.join.call(arguments, " ")]);
+        // We want to print out the stack trace when dynamics_nodejs.js throws
+        // an exception. Need to know what case this is and then make it into
+        // array so that we can pass it to console.log.apply.
+        else if (arguments.length === 2)
+          console.log.apply(console, [arguments[1]]);
```

Comment:
Ok, I can get rid of console.log.apply. I think it's bad practice, but I'm wondering what would be the best way to only know when we're in this specific error. Is it best to hardcode in another argument for the exception which is like 'print-stack-trace' and then pass in the stack trace and in only those cases print out the stack trace, thus not changing any current flow?